.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 7.1
```

Comment:
Why not 7.2? Because 7.3 already fails? We should then only check 7.2 (minimum requirement).